aw_watcher_window/main.py

```diff
@@ -92,10 +91,10 @@
                 poll_time=args.poll_time,
                 strategy=args.strategy,
                 exclude_title=args.exclude_title,
+                exclude_titles=[try: re.compile(title, re.IGNORECASE) except re.error: re.compile(re.escape(title), re.IGNORECASE) for title in args.exclude_titles]
```

Comment:
lol, Ellipsis really butchered this.

Reply:
hey @ErikBjare , looking into what happened here...

Reply:
@ErikBjare you can get much higher quality out of Ellipsis by adding a Dockerfile and build/test commands so that it can verify its changes - see docs here https://docs.ellipsis.dev/build. We have a playground as well where you can test it out at https://app.ellipsis.dev/

Reply:
happy to help get you onboarded to that  🙏